### PR TITLE
[no-jira]: Add test case for jetpack compose Prelaunch screen 

### PR DESCRIPTION
**Pull request by hadia**
# 📲 What

Increase test coverage by creating a Prelaunch screen test for jetpack compose 

# 🤔 Why
Increase test coverage

### CODE REVIEW

File: app/src/test/java/com/kickstarter/ui/activities/compose/PreLaunchProjectPageScreenTest.kt [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        projectImage.assertExists()
+        comingSoonBadge.assertExists()
+        projectName.assertTextEquals("")
+        projectName.assertIsNotDisplayed()
+        projectCreatorLayout.assertExists()
+        projectDescription.assertDoesNotExist()
+        projectCategory.assertDoesNotExist()
+        projectLocation.assertDoesNotExist()
+        projectSaveButton.assertExists()
+        projectSaveButton.assertTextEquals("Notify me on launch")
```
Comment (Arkariang) [inline app/src/test/java/com/kickstarter/ui/activities/compose/PreLaunchProjectPageScreenTest.kt:48]: This assert could be the actual string `R.id.Notify_me_on_launch`?

File: app/src/test/java/com/kickstarter/ui/activities/compose/PreLaunchProjectPageScreenTest.kt [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+                        projectState.value.watchesCount().toString()
+                    )
+                )
+            }
+        }
+
+        projectImage.assertExists()
+
+        comingSoonBadge.assertExists()
+        comingSoonBadge.assertTextEquals("Coming soon")
```
Comment (Arkariang) [inline app/src/test/java/com/kickstarter/ui/activities/compose/PreLaunchProjectPageScreenTest.kt:83]: Same for all of this asserts, could they be the resource ones? for example `R.strings.Coming_soon`

File: app/src/test/java/com/kickstarter/ui/activities/compose/PreLaunchProjectPageScreenTest.kt [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+import com.kickstarter.KSRobolectricTestCase
+import com.kickstarter.R
+import com.kickstarter.libs.graphql.DateTimeAdapter
+import com.kickstarter.mock.factories.ProjectFactory
+import com.kickstarter.models.Urls
+import org.junit.Test
+
+class PreLaunchProjectPageScreenTest : KSRobolectricTestCase() {
+
+    private val projectImage = composeTestRule.onNodeWithTag("Project Image")
```
Comment (Arkariang) [inline app/src/test/java/com/kickstarter/ui/activities/compose/PreLaunchProjectPageScreenTest.kt:21]: hey @hadia question here, what do you thing about having a companion object or some kind of structure on the on `PrelaunchProjectScreen` holding all the tags?

Comment (hadia) [reply app/src/test/java/com/kickstarter/ui/activities/compose/PreLaunchProjectPageScreenTest.kt]: Added enum class

**Pull request merged**